src/base/static/libs/maps/index.js

```diff
@@ -121,12 +121,29 @@ class MainMap {
       }
     });
 
-    // TEMPORARY: Manually trigger the visibility of a layer for testing
-    $(Shareabouts).trigger("visibility", [
-      this.mapConfig.layers[0].id,
-      true,
-      true,
-    ]);
+    // TEMPORARY: Manually trigger the visibility of layers for testing
+    this.map.on("load", () => {
+      $(Shareabouts).trigger("visibility", [
+        this.mapConfig.layers[3].id,
+        true,
+        true,
+      ]);
+      $(Shareabouts).trigger("visibility", [
+        this.mapConfig.layers[1].id,
+        true,
+        true,
+      ]);
+      $(Shareabouts).trigger("visibility", [
+        this.mapConfig.layers[0].id,
+        true,
+        true,
+      ]);
+      $(Shareabouts).trigger("visibility", [
+        this.mapConfig.layers[2].id,
+        true,
+        true,
+      ]);
+    });
```

Comment:
This will get replaced when we add a layer management UI. Eventually the layer UI will trigger these `visibility` events, but in the meantime we have to hard-code them to test individual layers.